Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_database.sql

```diff
@@ -0,0 +1,7 @@
+﻿DECLARE @dbname nvarchar(128)
```

Comment:
Do we still need this SQL script?

Reply:
@bsmithvt: What were we using it for originally? The integration tests use the sql generated from the dacpac to create the database, and we'll use the dacpac to deploy the db everywhere else.